<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" verbose="0" name="Failed suite [TestSuite1]">
  <parameter name="password" value="Pnbe@305"/>
  <parameter name="mobile" value="7799096333"/>
  <parameter name="email" value="anvardhan551@gmail.com"/>
  <listeners>
    <listener class-name="com.qa.app.listeners.RetryIAnnotationTransformerListener"/>
    <listener class-name="com.qa.app.listeners.ExtentReportListener"/>
  </listeners>
  <test thread-count="5" verbose="0" name="App1Test(failed)">
    <groups>
      <run>
        <include name="runall"/>
      </run>
    </groups>
    <classes>
      <class name="com.qa.tests.FinologyHomeTest">
        <methods>
          <include name="setUp"/>
          <include name="selectProduct_validateSelectUrl"/>
          <include name="tearDown"/>
          <include name="tickerProduct_validateTickerUrl"/>
        </methods>
      </class> <!-- com.qa.tests.FinologyHomeTest -->
    </classes>
  </test> <!-- App1Test(failed) -->
</suite> <!-- Failed suite [TestSuite1] -->
